### ISSUE TITLE
Missing rno-g import for reader

### ISSUE THREAD

**Issue by colemanalan** [nu-radio/NuRadioMC#581]
If I boot up the event browser, I get an import error at the following line

https://github.com/nu-radio/NuRadioMC/blob/7b5b259c2b06684b32f49c5610a6021e767cdadb/NuRadioReco/eventbrowser/dataprovider_root.py#L1

because firstly, that directory does not exist https://github.com/nu-radio/NuRadioMC/tree/develop/NuRadioReco/modules/io
 It might be that this is supposed to point to the `RNO_G` dir instead of `rno_g`, but even if this is the case, that file python is not there.

**Comment by fschlueter** [comment]
Yes, this is the old reader which we deprecate and did not care to implement the new one into the eventbrowser .... my bad!
